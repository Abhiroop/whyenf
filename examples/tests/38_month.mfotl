ALWAYS (FORALL x. A(x) IMPLIES EVENTUALLY[0,1M] B(x))