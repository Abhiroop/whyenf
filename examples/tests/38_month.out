version https://git-lfs.github.com/spec/v1
oid sha256:1766190ab1e08c8880af8cf47293704b12827af9a21d11c0e2434c323d0796ff
size 259200302
